KonyAccelerate/controllers/mobile/frmMoreController/frmMoreController.js

```diff
@@ -0,0 +1,73 @@
+define({ 
+
+ //Type your controller code here 
+  /** @function moreMenuAnimate
+   *  		    Slide in menu items one by one
+   */
+  animationDuration : 0.5,
+  menuAnimateSlideIn: function(){
+    var self = this;
+    let menuWidgets = this.view.flxMoreWrapper.widgets();
+    let delayThrashold = 0;
+    menuWidgets.forEach(function(currentMenu, index){
+      let currentItemIdex = index;
+      currentMenu.animate(
+        kony.ui.createAnimation({
+          0:{left:"-80%","stepConfig":{}},
```

Comment:
Can't we set the default value of left to "-80%" and just have the step for 100 here?

Reply:
Done